ui/gtk/main_window.py

```diff
@@ -63,12 +82,134 @@ def __init__(self, locale_general: LanguageManagement, config_general: dict, con
         self._component.get_component('main_window_header_right_label').set_label(
             u'<span font-size=\'x-large\'>{}</span>'.format(message))
 
+    def register_all_pages(self) -> None:
+        all_pages = list()
+
+        all_pages.append(LanguageLivePage(self._locale_general, self._config_general))
+        for current_page in all_pages:
+            current_page.load_win(self)
+            self._component.get_component('main_window_right_stack').add_named(
+                current_page._components.get_component('general_box'), current_page.get_page_id())
+            self._component.get_component('main_window_left_list_text_box').pack_start(
+                TitleLabel(current_page.get_page_id(), current_page.get_page_sidebar_title()), False, False, 0)
+            self._pages.append(current_page)
+
+    def load_lang(self) -> None:
+        component = self._component.get_component('main_window_left_list_text_box')
+        for label in component.get_children():
+            component.remove(label)
+
+        for current_page in self._pages:
+            label_in_box = TitleLabel(current_page.get_page_id(), current_page.get_page_sidebar_title())
+            if label_in_box.get_page_id() == self._pages[self._page_index].get_page_id():
+                label_in_box.get_style_context().remove_class('dim-label')
+            else:
+                label_in_box.get_style_context().add_class('dim-label')
+            self._component.get_component('main_window_left_list_text_box').pack_start(label_in_box, False,
+                                                                                       False, 0)
+            current_page.refresh_ui_language()
+        self._component.get_component('main_window_left_list_text_box').show_all()
+
+        self.set_title(self._pages[self._page_index].get_page_title())
+        self._component.get_component('main_window_prev_btn').set_label(
+            self._locale_general.translate_msg('main_window', 'bot_right_prev_btn'))
+        self._component.get_component('main_window_next_btn').set_label(
+            self._locale_general.translate_msg('main_window', 'bot_right_next_btn'))
+        self._component.get_component('main_window_header_right_quit_btn').set_label(
+            self._locale_general.translate_msg('main_window', 'top_right_quit_btn'))
+
+    def next_page_btn(self) -> None:
+        idx = self._page_index + 1
+        if idx >= len(self._pages):
+            return
+        self._page_index = idx
+        self.update_page()
+
+    def prev_page_btn(self) -> None:
+        idx = self._page_index - 1
+        if idx < 0:
+            return
+        self._page_index = idx
+        self.update_page()
+
+    def update_page(self) -> None:
+        self.set_title(self._pages[self._page_index].get_page_title())
+
+        if self._page_index == len(self._pages) - 1:
+            self.set_can_next(False)
```

Comment:
```suggestion
            self.set_can_next(self._page_index != len(self._pages) - 1)
```

(Same for all the `set_can_next`, `set_hide_next`, `set_can_prev`, `set_hide_prev` etc.)
(And obviously remove the outer condition)